lib/internal/abort_controller.js

```diff
@@ -53,10 +53,10 @@ function abortSignal(signal) {
   const event = new Event('abort', {
     [kTrustEvent]: true
   });
+  signal.dispatchEvent(event);
   if (typeof signal.onabort === 'function') {
```

Comment:
I think this should just be defined with defineEventHandler. I think the implementation (and the new one) aren't quite right in terms of ordering.

```js
const div = document.createElement('div');
div.addEventListener('click', () => console.log('listener 1'));
div.onclick = () => console.log('handler');
div.addEventListener('click', () => console.log('listener 2'));
div.dispatchEvent(new Event('click'))
```

Reply:
Will do. Thank you for your review!